assets/app/scripts/services/data.js

```diff
@@ -233,6 +319,30 @@ angular.module('openshiftConsole')
     }
     return deferred.promise;
   };
+  
+// name:      API name, the unique name for the object 
+// namespace: API namespace
+// url:       External URL
+// context:   API context, (e.g. { project: "..."})
+// opts:      force - always request (default is false)
+//            http - options to pass to the inner $http call
+  DataService.prototype.getTemplate = function(name, namespace, url, context, opts) {
```

Comment:
I'll remove this method actually... we're not going to allow remote templates for now. And yup, I agree a namespace parameter makes sense.